base_layer/wallet/src/connectivity_service/service.rs

```diff
@@ -194,8 +241,10 @@ impl WalletConnectivityService {
             "Successfully established peer connection to base node {}",
             conn.peer_node_id()
         );
-        let pool = conn.create_rpc_client_pool(self.config.base_node_rpc_pool_size);
-        self.base_node_wallet_rpc_client_pool = Some(pool);
+        self.pools = Some(ClientPoolContainer {
+            base_node_sync_rpc_client: conn.create_rpc_client_pool(self.config.base_node_rpc_pool_size),
+            base_node_wallet_rpc_client: conn.create_rpc_client_pool(self.config.base_node_rpc_pool_size),
```

Comment:
The `base_node_wallet_rpc_client` should have its own pool size configuration setting and not reuse the one from `base_node_sync_rpc_client`. Both pool size values should be present in the `config.toml` configuration file for case by case tuning. A busy console wallet needs to carry out much more than ` base_node_rpc_pool_size: 10,` conversations in parallel, specifically where the `base_node_wallet_rpc_client` is concerned.